prow/crier/reporters/gerrit/reporter_test.go

```diff
@@ -2194,6 +2195,38 @@ func TestMultipleWorks(t *testing.T) {
 	}
 }
 
+func TestJobReportFormats(t *testing.T) {
+	tests := []struct {
+		name        string
+		format      string
+		words       []interface{}
```

Comment:
these are strings

Reply:
> these are strings

In the future, can you explain what your suggestion is? Code review is most helpful when the review comments are as direct and unambiguous as possible.

I am going to assume that you'd like me to use `[]string` instead of `[]interface{}`. If so, doing a `[]string` to `[]interface{}` conversion for passing into `fmt.Sprintf()` seems like more trouble than it's worth. See https://stackoverflow.com/questions/27689058/convert-string-to-interface and https://stackoverflow.com/a/33418254

Reply:
ack